kcidb/orm/query.py

```diff
@@ -0,0 +1,830 @@
+"""
+The kcidb.orm.query module defines the query classes and functions used
+by the KCIDB object-relational mapping system to retrieve Kernel CI report
+data from the database.
```

Comment:
Same here, let's just write "The query classes ..." and so on. Otherwise, another nice one!

Reply:
Done!!!